### ISSUE TITLE
Can't stop video playback

### ISSUE THREAD

**Issue by kerenon** [mattermost/mattermost-mobile#8310]
#### Summary
Starting any kind of video (with sound) and pressing "X" or back to close it does not stop the video, the playback continues in the background (audio only). Tried multiple app versions (release + beta) and devices with the same results. The only option is to force close the mattermost application.

### Environment Information
  - Device Name: Pixel 7 Pro; OnePlus 6; Galaxy S21 Ultra
  - OS Version: Android 14, 11
  - Mattermost App Version: 2.22.0 (Build 6000569); 
  - Mattermost Server Version: 10.1.0 (Build 11150133805)

#### Steps to reproduce
Upload a video (with sound), open it, try to close with the "X" or the back button

#### Expected behavior
The video stops

#### Observed behavior (that appears unintentional)
The audio continues to play in the background, with no way to stop it

**Comment by amyblais** [comment]
Hi @kerenon, I wasn't able to reproduce it (I'm on iOS though). Are you able to reproduce this on our community server here https://community.mattermost.com/?

**Comment by kerenon** [comment]
Well, I tried testing it on the community server but joining the "Public Test Channel" crashes the app (release and beta).

![Screenshot_20241104-214502](https://github.com/user-attachments/assets/09f01ca3-5e77-4ee8-9ed4-819821a31796)

**Comment by enahum** [comment]
@hmhealey is the crash in this screenshot related to the tables change? Any ETA to have that fix in?

**Comment by kerenon** [comment]
@amyblais 
Managed to test it on the community server, all the same. The sound of the video just keeps going in the background without any way to stop it. This is not a server side problem.

**Comment by amyblais** [comment]
Opened https://mattermost.atlassian.net/browse/MM-61711.

**Comment by kerenon** [comment]
Yesterday I updatet my Pixel 7 to Android 15 and the problem is gone. It seems this is android version specific because all the other devices (Android 14 and below) still show the same problem.

**Comment by Zhoglik** [comment]
Same behavior.
Device - OnePlus 12 (Oxygen 14) and latest MM version FROM Play Market

**Comment by kuzmaMinin** [comment]
Hi! Ypu can fix it this way:

video_renderer/index.tsx



Before navigating to another screen  just pause the video

**Comment by lxxzhufu** [comment]
> Hi! Ypu can fix it this way:
> 
> video_renderer/index.tsx
> 
> ![Image](https://github.com/user-attachments/assets/777ee043-3a0c-4a6f-b131-d09d848b009b)
> 
> Before navigating to another screen just pause the video

Yes, it's okay to pause manually, but I don't think it's in line with user habits, when the video interface exits due to return, the program should automatically close the background video, instead of letting the background video play all the time
